bitcoin-information/technical-resources.html

```diff
@@ -120,6 +120,7 @@ <h2 id="resources"><a href="#resources">Technical Resources:</a></h2>
             <li><a href="http://www.samlewis.me/2017/06/a-peek-under-bitcoins-hood/" title="Under the Hood" target="_blank" rel="noopener">A Peek Under Bitcoin's Hood</a></li>
             <li><a href="https://codesuppository.blogspot.jp/2014/01/how-to-parse-bitcoin-blockchain.html" title="How to Parse the Blockchain" target="_blank" rel="noopener">How to Parse the Bitcoin Blockchain</a></li>
             <li><a href="https://bitcoinjs-guide.bitcoin-studio.com/" title="BitcoinJS Guide" target="_blank" rel="noopener">BitcoinJS &amp; Bitcoin Core CLI guide</a></li>
+            <li><a href="https://btclexicon.com" title="Bitcoin Glossary" target="_blank" rel="noopener">BTCLexicon.com</a></li>
```

Comment:
I think this is a better fit on the "getting started" page - I'd put it underneath the "frequently asked questions" link.

Reply:
Thank you, I moved BTCLexicon.com to "Getting Started" under "Frequently Asked Questions" (and left TimechainStats.com under Miscellaneous Metrics).